lib/topological_inventory/sync/sources_sync_worker.rb

```diff
@@ -104,13 +131,15 @@ def tenants_by_external_tenant(external_tenant)
         @tenants_by_external_tenant[external_tenant] ||= Tenant.find_or_create_by(:external_tenant => external_tenant)
       end
 
+      # TODO: Pagination not available in Sources Internal API!
```

Comment:
Does it just return all records then? If so that is probably fine.

Reply:
yeah, at least for next ~year. It will be problem with 100000+ of tenants

Reply:
ok, so eventually we will probably just need to add pagination to the /internal endpoints. Won't be bad as long as we stay on top of it.